actions/run-capslock/README.md

```diff
@@ -0,0 +1,30 @@
+# run-capslock
+
+Runs https://github.com/google/capslock action
+
+<!-- x-release-please-start-version -->
+
+```yaml
+name: Send And Respond to a Slack message using JSON payload
+jobs:
+  run-capslock:
+    name: Runs Capslock
+    steps:
+      - name: Run Capslock
```

Comment:
question: The action requires that the workspace already contains a repository checkout, right?